api_app/analyzers_manager/file_analyzers/malcore_scan.py

```diff
@@ -0,0 +1,60 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+
+class MalcoreScan(FileAnalyzer):
+    url = "https://api.malcore.io/api/upload"
+
+    _api_key_name: str
+
+    def run(self):
+        binary = self.read_file_bytes()
+        headers = {"apiKey": self._api_key_name}
+        files = {"filename1": binary}
+
+        try:
+            response = requests.post(self.url, headers=headers, files=files)
+            response.raise_for_status()
+        except requests.RequestException as e:
+            raise AnalyzerRunException(e)
+
+        result = response.json()
+        return result
```

Comment:
great but actually, as specified in the issue, this should be implemented like other similar sandbox services integrations we have in IntelOwl.
We expect this analyzer to:
* first check whether the file md5/sha256 has already been analyzed recently. In that case, just retrieve the report of the already existing analysis
* if that file is new, upload the file to the service like you just did here
* then, poll an api that checks the status of the analysis, until the analysis is ended
* once the analysis is ended, retrieve the final result

Please check other similar analyzers like [triage](https://github.com/intelowlproject/IntelOwl/blob/master/api_app/analyzers_manager/file_analyzers/triage_scan.py)